src/CardPage.js

```diff
@@ -1,24 +1,32 @@
 import React from 'react';
-import {useParams, Redirect} from 'react-router-dom';
 import {connect} from 'react-redux';
-import CardInfo from './CardInfo';
+import {useParams, Redirect} from 'react-router-dom';
+
 import {findCardById} from './appFunctions';
+import CardInfo from './CardInfo';
 
 function CardPage(props) { 
-    let {id} = useParams();           
-    let card = findCardById(id, props.cards);  
+    const {cards, statuses} = props;
+
+    const {id} = useParams();           
+    const card = findCardById(id, cards);  
     
     return (        
         card ? 
             <div>
-                <CardInfo id={card._id} />                   
+                <CardInfo 
+                    id={card._id}
+                    cards={cards}
+                    statuses={statuses}
+                />                   
             </div>
         : <Redirect to="/" />                           
     )    
 }
 
 const mapStateToProps = (state) => ({
-    cards: state.cards.data
+    cards: state.cards.data,
+    statuses: state.statuses.data
```

Comment:
is it necessary to have connect here? don't you have any parent that use cards or statuses props?

Reply:
App.js render CardPage or HomePage.
It is two separate independing Page.
We discussed this